<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Statistical Coupling Analysis</title>
    <link rel="stylesheet" href="style.css" />
    <link rel="stylesheet" href="mediaqueries.css" />
  </head>
  <body>
    <nav id="desktop-nav">
        <div class="logo">Ethan Risman</div>
        <div>
          <ul class="nav-links">
            <li><a href="https://ethanrisman.github.io/About/">About</a></li>
            <li><a href="https://ethanrisman.github.io/About/#experience">Experience</a></li>
            <li><a href="https://ethanrisman.github.io/About/#projects">Projects</a></li>
            <li><a href="https://ethanrisman.github.io/About/#contact">Contact</a></li>
          </ul>
        </div>
      </nav>
      <nav id="hamburger-nav">
        <div class="logo">Ethan Risman</div>
        <div class="hamburger-menu">
          <div class="hamburger-icon" onclick="toggleMenu()">
            <span></span>
            <span></span>
            <span></span>
          </div>
          <div class="menu-links">
            <li><a href="https://ethanrisman.github.io/About/"onclick="toggleMenu()">About</a></li>
            <li><a href="https://ethanrisman.github.io/About/#experience"onclick="toggleMenu()">Experience</a></li>
            <li><a href="https://ethanrisman.github.io/About/#projects"onclick="toggleMenu()">Projects</a></li>
            <li><a href="https://ethanrisman.github.io/About/#contact"onclick="toggleMenu()">Contact</a></li>
          </div>
        </div>
    </nav>
    <section class="sub-page">
        <div class="large-text">What is statistical coupling analysis?</div>
        <div class="medium-text">
          SCA offers the potential to identify secondary residues by identifying correlations between changes in protein sequences. For instance, in a sequence alignment, residues that are not highly conserved may initially seem insignificant. However, by investigating the connectivity of these residues, SCA can reveal interesting relationships between them. pySCA, which is the script that runs SCA, was created by Lockless, Rivoire, Reynolds, and Ranganathan.
        </div>
      
        <div class="picture-and-caption">
          <div class="SCA-photo">
            <img src="./assets/project-1.png" alt="project picture" />
          </div>
          <div class="photo-caption">
            <p style="color: rgb(0, 0, 0);"><b>Toy diagram of a SCA read.</b> 
              <b>A</b> Configuration where Asp and Arg form a salt bridge (green). Configuration where Asp and Lys form a salt bridge (blue).
              <b>B</b> Conservation of either sequence is 50% but coupling dynamics are seen 100% of the time.
            </p>
          </div>
        </div>

        <div class="large-text">Where can SCA be used?</div>
        <div class="medium-text">
            <p>
                pySCA is best used on conserved protein homologs. For example, terpene synthases, transcription factors, or structural proteins. Each species has its own version, and each version will exibit different characteristics. 
                <br /><br />
                Lets take a closer look at pySCA, but running it on an assortment of RuBisCo proteins from varied planta.
                <br /><br />
                Well I would take a closer look, but I foolishly installed the new OS... so now we wait. 
            </p>
        </div>
      </section>
  </body>